go.mod

```diff
@@ -292,4 +292,4 @@ replace github.com/opentracing-contrib/go-stdlib => github.com/grafana/opentraci
 replace github.com/opentracing-contrib/go-grpc => github.com/charleskorn/go-grpc v0.0.0-20231024023642-e9298576254f
 
 // Replacing prometheus/alertmanager with our fork.
-replace github.com/prometheus/alertmanager => github.com/grafana/prometheus-alertmanager v0.25.1-0.20240605141526-70d9d63f74fc
+replace github.com/prometheus/alertmanager => github.com/grobinson-grafana/alertmanager v0.0.0-20240620142649-edab848b6cee
```

Comment:
Needs to be changed as soon as https://github.com/grafana/prometheus-alertmanager/pull/87 is merged.